tests/unit/controlgroup/controlgroup_options.js

```diff
@@ -0,0 +1,125 @@
+module( "Controlgroup: options" );
+
+function hasCornerClass( className, element ) {
+	if ( className ) {
+		return element.hasClass( className );
+	}
+
+	return element.attr( "class" ).match( /ui-corner/g );
+}
+
+test( "disabled", function() {
```

Comment:
Same problems as the disable/enable method tests, should get the same treatment.